web/src/compose_banner.ts

```diff
@@ -52,9 +52,9 @@ export function clear_message_sent_banners(): void {
 
 // TODO: Replace with compose_ui.hide_compose_spinner() when it is converted to ts.
 function hide_compose_spinner(): void {
-    $("#compose-send-button .loader").hide();
-    $("#compose-send-button span").show();
-    $("#compose-send-button").removeClass("disable-btn");
+    $(".compose_message_button .loader").hide();
+    $(".compose_message_button span").show();
+    $(".compose_message_button").removeClass("disable-btn");
```

Comment:
Maybe `compose-submit-button` would be a better name? I think that would make the role clear. (I think we're also preferring `-`s over `_`s for separators in CSS).

Reply:
Updated it.